### ISSUE TITLE
RuntimeError: CUDA out of memory.

### ISSUE THREAD

**Issue by g2502823040** [dvlab-research/UVTR#4]
您好，我想问一下该项目使用的GPU类型，我用的是TITAN RTX，但在使用projects/configs/uvtr/camera_based/knowledge_distill/uvtr_l2cs3_r101_h11.py跑训练的时候，会出现显存不够。
RuntimeError: CUDA out of memory. Tried to allocate 408.00 MiB (GPU 0; 23.65 GiB total capacity; 5.46 GiB already allocated; 212.12 MiB free; 5.95 GiB reserved in total by PyTorch)

**Comment by yanwei-li** [comment]
Hi, most of the experiments are trained with V100. For memory costing settings, like multi-frame with R101 (this one) or multi-modality, we use A100 for training.

**Comment by sujinjang** [comment]
First of all, thanks for sharing great work. I also have a question on a cuda memory issue in training. I tried to run an experiment with the same config "uvtr_l2c23_r101_h11.py" on A100, but got the following issue:
"CUDA out of memory. Tried to allocate 12.38 GiB (GPU 1; 79.35 GiB total capacity; 61.38 GiB already allocated; 1.97 GiB free; 61.66 GiB reserved in total by PyTorch" 
How did you run it on A100 gpus? Am I missing something here?
